app/server/routes/api.js

```diff
@@ -403,6 +403,41 @@ module.exports = function(router) {
     UserController.sendWaiverEmail(id, defaultResponse(req, res));
   });
 
+  /**
+   * Add user to acceptance queue.
+   */
+  router.post('/users/:id/queue', isAdmin, function(req,res){
+    var id = req.params.id;
+    UserController.addUserAcceptedQueue(id, defaultResponse(req, res));
+  });
+
+  /**
+   * Remove user from acceptance queue.
+   */
+  router.delete('/users/:id/queue', isAdmin, function(req,res){
+    var id = req.params.id;
+    UserController.removeUserAcceptedQueue(id, defaultResponse(req, res));
+  });
+
+  /**
+   * Accept all users in acceptance queue
+  */
+  router.post('/users/queue', isAdmin, function(req,res){
```

Comment:
Rename this route. It is confusing since this route sounds like it should let me view the queue. Maybe trying something like "acceptall" or "acceptqueue"